### ISSUE TITLE
Compatibility with TYPO3 7.3 and up

### ISSUE THREAD

**Issue by leonderijke** [beechit/fal_securedownload#16]
In TYPO3/TYPO3.CMS@1beb07f03e0dd9316f8bb96d8ef2a75803cda69d a change was introduced, making Condition ViewHelpers fully static compilable. As far as I can tell, the `AssetAccessViewHelper`'s `render` method is now only executed once in an empty cache situation. After that, the `renderStatic` method (from the `AbstractConditionViewHelper`) is executed. In it, the static method `evaluateCondition` is then used to determine whether to render the `then` or the `else` child.

`evaluateCondition` checks for the `condition` argument, which is not present when using the `AssetAccessViewHelper`, causing the `else` child to render in a cached situation.

Reproducable with this code in the `FileTree/Tree.html` template:
```xml
<sd:security.assetAccess folder="{folder}">
  <f:then>This is rendered the first time</f:then>
  <f:else>This is rendered the second time and beyond</f:else>
</sd:security.assetAccess>
```

**Comment by fsaris** [comment]
Some background about the source of this issue http://stackoverflow.com/questions/34729014/typo3-ver-7-6-2-condition-viewhelpers-evaluated-only-once/34743478#34743478

**Comment by fsaris** [comment]
Fixed with af7179c0848beddb4351efb84ea0b6da9301b451
